docs/user/setup.asciidoc

```diff
@@ -21,6 +21,9 @@ Windows. Since Kibana runs on Node.js, we include the necessary Node.js
 binaries for these platforms. Running Kibana against a separately maintained
 version of Node.js is not supported.
 
+To support certain older Linux platforms (most notably CentOS7/RHEL7), {kib}
```

Comment:
I removed a link to fix the docs build - see https://github.com/elastic/kibana/pull/160289/files#diff-aada4c76a198b66a50968538ca0a3cfa7581cff465a3819392de368dd8ed9940R34.  The linked section is part of our developer docs that doesn't exist on 7.17.

Open to adding the advanced section in but kept it simple to start.